### PR TITLE
Added Japanese localization.

### PR DESCRIPTION
**Pull request by zombine04**

### CODE REVIEW

File: dmf/localization/dmf.lua [hunk 1]
```diff
@@ -283,24 +330,30 @@ return {
     ["zh-cn"] = "应该删除哪些重复记录。\n\n" ..
          "-- 最新 --\n如果和最新一条匹配，则删除上一条记录。\n\n" ..
          "-- 所有 --\n如果和最新一条匹配，则删除所有匹配的记录。",
+    ja = "重複した際にどの履歴を削除するか。\n\n" ..
+         "-- 直前 --\n1つ前の履歴が重複する場合、それを削除します。\n\n" ..
```
Comment (sclu1034) [inline dmf/localization/dmf.lua]: The `1` after the line break looks out-of-place.

**Review by sclu1034 — CHANGES_REQUESTED** [review]

File: dmf/localization/dmf.lua (hunk 1 above)
Comment (zombine04) [reply dmf/localization/dmf.lua]: I thought `previous entry` means just one before, so I translated it to `1つ前`. Is that incorrect?

Comment (zombine04) [reply dmf/localization/dmf.lua]: Any way, I've updated the translation to match its title.

Comment (sclu1034) [reply dmf/localization/dmf.lua]: > I thought `previous entry` means just one before, so I translated it to `1つ前`. Is that incorrect?

If you do normally use literal numbers for this in Japanese, that's fine.
I guess I'm just used to languages having a dedicated word for "previous" and assumed that the `1` might just be an unrelated typo.

Comment (zombine04) [reply dmf/localization/dmf.lua]: It was not a typo, but it's fine without the `1`.

**Pull request merged**